libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -184,6 +189,20 @@ pair<string, set<string>> ABIFunctions::requestedFunctions()
 	return make_pair(result, std::move(m_externallyUsedFunctions));
 }
 
+string ABIFunctions::EncodingOptions::toFunctionNameSuffix() const
+{
+	string suffix;
+	if (!padded)
+		suffix += "_nonPadded";
+	if (dynamicInplace)
+		suffix += "_inplace";
```

Comment:
Nitpicking: these should be `dynamicInPlace` and `_inPlace` if we follow `nonPadded`.